allcontainers.js

```diff
@@ -98,7 +105,17 @@ function allContainers (opts) {
         return result.emit('error', err)
       }
 
-      containers.forEach(emit)
+      var activeContainers = [];
+      function findPausedContainers(container) {
+        if (container.Status.indexOf("Paused") !== -1) {
+          pausedPreheatContainers[container.Id] = container;
+        } else {
+          activeContainers.push(container);
+        }
+      }
```

Comment:
please define this function outside the through scope, and use a `reduce`  operator to populate the `activeContainers`  array.